docker/docker-compose.yml

```diff
@@ -59,6 +59,9 @@ services:
     hostname: elasticsearch
     ports:
       - "9200:9200"
+    environment:
```

Comment:
Make sure to run https://github.com/linkedin/datahub/blob/master/docker/quickstart/generate_docker_quickstart.sh script so the quickstart gets updated as well